### ISSUE TITLE
Users created via user.props are not showing in users tab but auth is working

### ISSUE THREAD

**Issue by pavanmallepudi** [datahub-project/datahub#4518]
I have added below contents in user.props file and using helm I can see contents also from secrets 
admin:admin
user:user

 kubectl exec datahub-datahub-frontend-58599c45fd-f24mq -- cat /etc/datahub/plugins/frontend/auth/user.props
admin:admin
user:user

But in ui these users are showing while configuring policies . Kindly help me if I need to add more details in user.props

**Comment by maggiehays** [comment]
@pedro93 please take a look at this one!

**Comment by pedro93** [comment]
Hello Pavan!

Users added through the `user.props` file are used only for **authentication** purposes (can a user log in or not). This is different from adding users to datahub's metadata graph. 

You may want to create a visitor/robot account which has no interaction within DataHub. It can login into datahub but is not part of the set of users that interact with the several parts of the metadata model, i.e: datasets, charts, models, etc...

Datahub provides 2 authentication methods:
1.) JAAS -> Java Authentication and Authorization Service
2.) OIDC -> OpenID connect

**Comment by jjoyce0510** [comment]
@pavanmallepudi Did this help? Please ping on us Slack if you need more help here. Want to make sure you are unblocked.

**Comment by jjoyce0510** [comment]
Hi @pavanmallepudi - We've recently added support for creating new users via the UI. Check out https://datahubproject.io/docs/authentication/guides/add-users.

I'm going to close this issue since it should be resolved now.

Thanks
John
